overlays/jaeger/jaeger.Deployment.yaml

```diff
@@ -45,6 +45,10 @@ spec:
               protocol: TCP
             - containerPort: 14250
               protocol: TCP
+            - containerPort: 4320
+              protocol: TCP
+            - containerPort: 4320
+              protocol: TCP
```

Comment:
duplicate ports?

Reply:
Maybe we can choose just to expose one (the gRPC one)

Reply:
Great catch!